### PR TITLE
Oversized recipes and clothing items

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary

SUMMARY: Content "Oversize recipes and clothing"

#### Purpose of change

Adding oversized items on the scratch made list.  

#### Describe the solution

Originally added an Oversized Costuming Proficiency but eso and mlangsdorf walked me through what they feel is a better near term direction.

#### Describe alternatives you've considered

Most alternatives discussed would involve recreating all aspects of tailoring it seems.

#### Testing
I've been testing this is in my own game and haven't seen any errors yet.

### CODE REVIEW

File: data/json/items/armor/arms_armor.json [hunk 1]
```diff
@@ -147,6 +198,16 @@
     "environmental_protection": 1,
     "flags": [ "BELTED", "BLOCK_WHILE_WORN", "WATER_FRIENDLY" ]
   },
+  {
+    "id": "xl_armguard_metal",
+    "type": "ARMOR",
+    "name": { "str": "pair of oversize metal arm guards", "str_pl": "pairs of oversize metal arm guards" },
+    "description": "A pair of arm guards hammered out from metal.  Perfect for the Post-Apocalyptic Warrior look. Larger than average.",
```
Comment (LaVeyanFiend) [inline data/json/items/armor/arms_armor.json:205]: Extra space before "Larger" to fix error.

File: data/json/items/armor/arms_armor.json [hunk 2]
```diff
@@ -280,6 +371,16 @@
     "environmental_protection": 1,
     "flags": [ "STURDY", "BELTED", "WATER_FRIENDLY" ]
   },
+  {
+    "id": "xl_vambrace_larmor",
+    "type": "ARMOR",
+    "name": { "str": "pair of xl leather vambraces", "str_pl": "pairs of xl leather vambraces" },
+    "description": "Customized oversized leather vambraces, made for archery. ",
```
Comment (LaVeyanFiend) [inline data/json/items/armor/arms_armor.json:378]:
```suggestion
    "description": "Customized oversized leather vambraces, made for archery.",
```

File: data/json/recipes/armor/feet.json [hunk 3]
```diff
@@ -69,6 +89,16 @@
       { "proficiency": "prof_articulation", "required": false, "time_multiplier": 2, "fail_multiplier": 1.5 }
     ]
   },
+  {
+    "result": "xl_boots_acidchitin",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
```
Comment (LaVeyanFiend) [inline data/json/recipes/armor/feet.json:97]:
```suggestion
```

File: data/json/recipes/armor/feet.json [hunk 4]
```diff
@@ -49,6 +59,16 @@
       { "proficiency": "prof_articulation", "required": false, "time_multiplier": 2, "fail_multiplier": 1.5 }
     ]
   },
+  {
+    "result": "xl_boots_chitin",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
```
Comment (LaVeyanFiend) [inline data/json/recipes/armor/feet.json:67]:
```suggestion
```

File: data/json/recipes/armor/feet.json [hunk 5]
```diff
@@ -30,6 +30,16 @@
       { "proficiency": "prof_leatherworking", "required": false, "time_multiplier": 2 }
     ]
   },
+  {
+    "result": "xl_boots",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
```
Comment (LaVeyanFiend) [inline data/json/recipes/armor/feet.json:38]:
```suggestion
```

File: data/json/recipes/armor/feet.json [hunk 6]
```diff
@@ -89,6 +119,16 @@
     ],
     "components": [ [ [ "duct_tape", 50 ] ], [ [ "boots_combat", 1 ], [ "boots_steel", 1 ], [ "boots_bunker", 1 ] ] ]
   },
+  {
+    "result": "xl_boots_fsurvivor",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
```
Comment (LaVeyanFiend) [inline data/json/recipes/armor/feet.json:127]:
```suggestion
```

**Review by LaVeyanFiend — CHANGES_REQUESTED** [review]

File: data/json/items/armor/hoods.json [hunk 7]
```diff
@@ -43,6 +43,15 @@
     "material_thickness": 4,
     "flags": [ "VARSIZE", "STURDY", "HELMET_COMPAT" ]
   },
+  {
+    "id": "xl_chainmail_hood",
+    "type": "ARMOR",
+    "copy-from": "chainmail_hood",
+    "name": { "str": "oversize chainmail coif" },
+    "description": "A customized chainmail hood.  Can be worn comfortably under helmets. Comes in size XL.",
```
Comment (Maleclypse) [inline data/json/items/armor/hoods.json:51]:
```suggestion
    "description": "A customized chainmail hood.  Can be worn comfortably under helmets.  Comes in size XL.",
```

File: data/json/items/armor/hoods.json [hunk 8]
```diff
@@ -67,6 +76,15 @@
     "environmental_protection": 10,
     "flags": [ "VARSIZE", "WATERPROOF", "STURDY", "OUTER", "HELMET_COMPAT" ]
   },
+  {
+    "id": "xl_hood_fsurvivor",
+    "type": "ARMOR",
+    "copy-from": "hood_fsurvivor",
+    "name": { "str": "oversize survivor firehood" },
+    "description": "A customized armored Nomex and Kevlar hood, very strong and durable. Made in Muspelheim.",
```
Comment (Maleclypse) [inline data/json/items/armor/hoods.json:84]:
```suggestion
    "description": "A customized armored Nomex and Kevlar hood, very strong and durable.  Made in Muspelheim.",
```

File: data/json/recipes/armor/suit.json [hunk 9]
```diff
@@ -1038,6 +1230,32 @@
       [ [ "kevlar", 1 ], [ "ballistic_vest_empty", 1 ], [ "swat_armor", 1 ], [ "sheet_kevlar_layered", 24 ] ]
     ]
   },
+  {
+    "result": "xl_wsurvivor_suit",
+    "copy-from": "wsurvivor_suit",
+    "time": "8 h",
```
Comment (Maleclypse) [inline data/json/recipes/armor/suit.json:1236]:
```suggestion
    "time": "8 h",
    "type": "recipe",
```

**Comment by Maleclypse** [comment]
@ZhilkinSerg Is there anything you need me to do on this PR for it to get merged?

**Pull request merged**